mobile/src/Register.tsx

```diff
@@ -6,51 +6,61 @@ import {
   Button,
   SafeAreaView,
   StyleSheet,
+  Keyboard,
+  TouchableWithoutFeedback,
+  ActivityIndicator,
 } from "react-native";
 import { Formik } from "formik";
 import { register } from "./authSlice";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import sharedStyles from "./sharedStyles";
 
 const Register: React.FC<{}> = () => {
   const dispatch = useDispatch();
+  const { loading, error } = useSelector((state) => state.auth);
 
   return (
-    <SafeAreaView style={sharedStyles.container}>
-      <Formik
-        initialValues={{ email: "", password: "" }}
-        onSubmit={(values) => {
-          dispatch(register(values));
-          console.log(values);
-        }}
-      >
-        {({ handleChange, handleBlur, handleSubmit, values }) => (
-          <View>
-            <Text style={styles.heading}>Register</Text>
-            <TextInput
-              onChangeText={handleChange("email")}
-              onBlur={handleBlur("email")}
-              value={values.email}
-              placeholder="email"
-              style={sharedStyles.input}
-            />
-            <TextInput
-              onChangeText={handleChange("password")}
-              onBlur={handleBlur("password")}
-              value={values.password}
-              placeholder="password"
-              style={sharedStyles.input}
-              secureTextEntry
-            />
-            <Button
-              accessibilityLabel="Submit"
-              onPress={handleSubmit}
-              title="Submit"
-            />
-          </View>
-        )}
-      </Formik>
-    </SafeAreaView>
+    <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
```

Comment:
I tried searching on docs, but what exactly does this do?

Reply:
When you tap anywhere outside of the form fields, this dismisses the keyboard. The TouchableWithoutFeedback is basically a big invisible button that surrounds the component that captures the press event when it's not in the email or password field. This is so if you want to go to another page or scroll around or whatever, you can do so without the keyboard being in the way.

Reply:
I understand. https://reactnative.dev/docs/touchablewithoutfeedback/ says do not use TouchableWithoutFeedback unless you have a very good reason & to check out Pressable API for a " a more extensive and future-proof way to handle touch-based input". Maybe we can incorporate Pressable in the future!

Reply:
Sounds good. Yeah we don't really need any feedback on exiting out of the fields so this is what I'm used to using. The Pressable component looks like its more fully featured, so makes sense to use that in the future.